karate-tests/src/test/java/patients/healthcareWorker/updatePatientDetails/addAndRemove.feature

```diff
@@ -184,3 +185,56 @@ Feature: Patch patient - Add and remove data
     * status 200
     * match response.name[0].suffix == '#notpresent'
     * match parseInt(response.meta.versionId) == originalVersion + 1
+
+
+  Scenario:  Healthcare worker can add and remove place of birth details(city and district)
+    * def placeOfBirthNhsNumber = '5900077810'
+    * configure headers = call read('classpath:auth/auth-headers.js') 
+    * path 'Patient', placeOfBirthNhsNumber
+    * method get
+    * status 200
+    * def originalVersion = parseInt(response.meta.versionId)
+
+    #add place of birth details
```

Comment:
Can you add a place of birth if there is already one on the record? If not then there's a possibility of the test failing on the remove stage, and will then fail every time from then on. If my assumptions right, can you add the expected state the record should be in at the top of the scenario?

Reply:
Added a step to terminate the test if place of birth is not on the response